libraries/engage/tracking/streams/index.js

```diff
@@ -0,0 +1,27 @@
+import { main$ } from '@shopgate/pwa-common/streams';
+import { UPDATE_COOKIE_CONSENT } from '../constants';
+
+/**
+ * Gets triggered when the cookie consent has been handled by the user.
+ * @type {Observable}
+ */
+export const cookieConsentUpdated$ = main$.filter(({ action }) => (
```

Comment:
those streams are supposed to be used run code after the cookie consent step is finished - independent of user actions. 

right now the stream is based on the `UPDATE_COOKIE_CONSENT` action which is only dispatched during interaction with the modal. this action always mutates the `cookieSettings` state, so it can't be dispatched with `true` for both properties when the cookie consent feature is inactive.

Reply:
fixed